src/amo/components/Addon/index.js

```diff
@@ -219,6 +220,8 @@ export class AddonBase extends React.Component {
             {...this.props}
             className="Button--action Button--small"
             disabled={!compatible}
+            ref={(ref) => { this.installButton = ref; }}
```

Comment:
Yup, in the tests https://github.com/mozilla/addons-frontend/pull/2554/files#diff-25c04fb93f8e7aaba2269e69a7182c11R328